Directory.Build.targets

```diff
@@ -1,5 +1,6 @@
 <Project>
   <PropertyGroup>
+    <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">$(MicrosoftNETCoreApp21PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
What's still using netcoreapp2.1?

Reply:
ResponseCompression cross compiles for netcoreapp2.1. Kestrel does something similar.

Reply:
Why? This is going into the 2.2 release, right? I think it should be lifted to target netcoreapp2.2.

Reply:
Product packages are staying at minimum supported TFMs.

Reply:
There is no point in having this target netcoreapp2.1. We would not support mixing this package in with 2.1 dependencies. Because this package has overlapping dependencies with Microsoft.AspNetCore.App, it wouldn't work with Microsoft.AspNetCore.App anyways.